src/edu/stanford/nlp/ling/tokensregex/CoreMapExpressionExtractor.java

```diff
@@ -136,11 +138,16 @@ public CoreMapExpressionExtractor(Env env) {
     this.tokensAnnotationKey = EnvLookup.getDefaultTokensAnnotationKey(env);
     if (env != null) {
       this.collapseExtractionRules = Objects.equals((Boolean) env.get("collapseExtractionRules"), true);
+      this.keepNestedMatches = Objects.equals((Boolean) env.get("keepNestedMatches"), true);
```

Comment:
The truth is, I know almost nothing about tokensregex, and I would say the people still at Stanford are generally in the same boat.  Why is it necessary to use `Objects.equals` here?  I see that it was done for a different `Boolean` earlier as well

Reply:
also, should the default be false instead?  that way the behavior doesn't change for existing use cases